samples-js/functions/httpStart.js

```diff
@@ -0,0 +1,18 @@
+const df = require("durable-functions");
+const { app } = require("@azure/functions");
+
+const clientInput = df.input.durableClient();
+
+app.http("httpStart", {
+    route: "orchestrators/{orchestratorName}",
+    extraInputs: [clientInput],
+    handler: async (request, context) => {
+        const client = df.getClient(context, clientInput);
+        const body = await request.json();
+        const instanceId = await client.startNew(request.params.orchestratorName, undefined, body);
```

Comment:
We won't need this `undefined` parameter in the future, right?

Reply:
No we don't, pending #429. That being said, my general philosophy towards migrating these samples is that it is a pretty big task and a lot of things can break, so instead of waiting until _all_ the changes in the new prog model are ready, let me just get the bulk of the work done now, and have a working app with the current public release of the SDK, and then let's go back and update the samples once we have some of those updates in. Let me know if you have other thoughts or if this makes sense, but hopefully this addresses some of your other comments too

Reply:
I just updated this to the new public release `3.0.0-alpha.2` and updated this line too.